src/WWT.Providers/IWwtContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Web;
+
+namespace WWT.Providers
+{
+    public interface IWwtContext
```

Comment:
introducing this abstraction is a really good idea, I think this sets us up for a lot more confidence in future refactoring efforts